railties/lib/rails/generators/rails/encrypted_secrets/templates/config/secrets.yml.enc

```diff
@@ -0,0 +1,2 @@
+production:
+  secret_key_base: <%= SecureRandom.hex(64) %>
```

Comment:
Got it, thanks! So all I need to do is provide a `RAILS_MASTER_KEY` instead of a `RAILS_SECRET_KEY_BASE`. Can do.